### ISSUE TITLE
More premium pricing plans

### ISSUE THREAD

**Issue by natedotlife** [sybrew/the-seo-framework#87]
Hey there!

I love the SEO Framework and have been using it since release, but I'm facing a big barrier to purchasing premium with the pricing models you currently have. I'm not willing to spend $250 a year and I have no need for x100 licenses, but I'm also not willing to spend $150 a year for ~6 sites (I need about that many licenses at present). Intermittently, I haven't purchased, even though I'd love to be in the early bird club and would love to support the plugin.

Would you consider an "agency" option that is middle ground? Maybe 10-12 keys for $80-$100/ year? I imagine there are a lot of on-the-fencers like myself who would pull the trigger were that the case.

Thanks~

**Comment by LeBaux** [comment]
I was thinking about this a little too, and pricing is always an issue where you can't satisfy everyone, because:

1. **Geographical economic differences** - something that is affordable in USA, is considered expensive in 80% other countries.
2. **Overall business viability** - Sadly, tons of statups admitted that it is more viable to have more expensive tiers, because at the end of the day lot of people wants the cheaper option, but only small number will actually pay. This conveys to reason 3.
3. **Value** - no matter how cheap premium will be, if the free offers enough lots of people will just never pay.

Good thing is currently you have everything you need for free, and premium is really just cream on top. My personal favorite here would be flat pricing at 1usd/month/site. It you have 3, 5, 10, 25, 50, 100, 250, 500, 1000 there could be discount of 5, 10, 15, 20, 25, 30, 40, 50, 60%. While this would be considered affordable in most Europe counties, it is just dirty cheap for USA.

**Comment by sybrew** [comment]
Hi guys!

This is in consideration! 

Currently, it's all in its very early stages and I *really* should improve the website's layout, because if I were to add more pricing options it would become *even more* cloudy.

Also, more choices will certainly become confusing for most users. So I need to find the right approach (i.e. dropdown selection).

Unfortunately, I can't promise that these additions will fall in the Q1 or even the any of the Early Bird pricing categories. There's just too much work to be done to realize this on my own; certainly because we're dealing with lifetime discounts.

**Comment by NateDotLife** [comment]
> Unfortunately, I can't promise that these additions will fall in the Q1 or even the any of the Early Bird pricing categories. There's just too much work to be done to realize this on my own; certainly because we're dealing with lifetime discounts.

Great to hear you're thinking about it! I'd strongly suggest you prioritize this before the honeymoon period ends, less from a selfish POV (though there's that :D) and more from a marketing sciences POV:

At the moment you're pushing a **launch** of a premium product. Your Q1/Q2/Q3 schema is effectively marketing to/ rewarding true "early adopters" in your marketing mix. The LTV of your early adopters is guaranteed to be more than _any other customer_ you convert down the line, as this segment is statistically much more inclined to engage -- [check out this study](http://tech.co/your-early-adopters-should-be-your-initial-marketing-strategy-2013-01). As an example, I have personally evangelized TSF and it is running on some major media/ eCom sites because of my suggestions. I'm also willing to write essays of candid feedback.

If your early adopters are pushed away by the _first_ pricing barrier, you have a make-it-or-break-it problem... they are much less likely, in economic terms, to return later after the price has increased, especially in such a noisy space as SEO tools, because they've already seen earlier pricing. Those who are so vested in TSF to want to purchase Premium now, even when the premium roadmap is rather thin and still shakily defined, are lost evergreen revenue.

To put it simply --

Yes, you'll earn less by diversifying your Early Bird pricing _now_, but in terms of long run value, these are your **most valuable customers**. If you don't convert them, you'll never have another chance (early adopters don't return), and you're losing out on hundreds or thousands of LTV $$$ down the line.

**Comment by sybrew** [comment]
Hi @NateDotLife,

After much consideration, I will prioritize the variable pricing additions 😄
I'll need to add a compatibility layer in the API server and some theme adjustments, so it will require some time.

Thanks for all your valuable input! Cheers!

**Comment by natedotlife** [comment]
Happy to hear it man :) Looks like you'll have at least two sales right off the bat!

**Comment by sybrew** [comment]
Done & Done 👍 

https://premium.theseoframework.com/shop/ 

Enjoy 😄
